pyuvdata/beamfits.py

```diff
@@ -128,8 +128,13 @@ def read_beamfits(self, filename, run_check=True, check_extra=True,
                 self.data_array = np.squeeze(complex_arrs[0] + 1j * complex_arrs[1], axis=0)
             else:
                 self.data_array = data
+
+            # Note: This axis is called STOKES by analogy with the equivilent uvfits axis
+            # However, this confusing because it is NOT a true Stokes axis,
```

Comment:
Couple typos: equivilent -> equivalent; "this confusing" -> "this is confusing"

Reply:
thanks, fixing them now